packages/generator-one-app-module/generators/app/templates/base-module/src/config.js

```diff
@@ -0,0 +1,5 @@
+import csp from './csp';
+
+export default {
```

Comment:
We should probably add a comment explaining what this config is doing and why. Perhaps link to the module config docs in the one app readme?

Reply:
This would be the same link as what's provided above the `global.BROWSER` if block that houses the appConfig. The One App docs give an example where all the logic (csp, provideStateConfig, CorsOrigins, etc) is declared in the same file, rather than importing it from a separate `config.js` file. 

I think it'd be better for these to be consistent and for the generator to mimic what's in the docs example. I'm gonna remove the separate `config.js` file and put the configurations directly within the if block.